pkg/scheduler/framework/plugins/noderesources/resource_limits_test.go

```diff
@@ -149,12 +157,20 @@ func TestResourceLimits(t *testing.T) {
 			}
 			for i := range test.nodes {
 				hostResult, err := p.Score(context.Background(), state, test.pod, test.nodes[i].Name)
+				if test.skipPostFilter {
```

Comment:
we don't seem to be skipping post filter.

Reply:
done.